languages/ruby/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,87 @@
+One of the key aspects of working with numbers in Ruby is the distinction between integers (numbers with no digits after the decimal separator) and floating-point numbers (numbers with zero or more digits after the decimal separator).
+They are implemented through the [`Integer`][integer-ruby] and [`Float`][float-ruby] class.
+
+```ruby
+a = 1
+b = 1.0
+a.class
+#=> Integer
+b.class
+#=> Float
```

Comment:
Maybe use descriptive names and group the class calls with the definitions?

```suggestion
integer_variable = 1
integer_variable.class
#=> Integer

float_variable = 1.0
float_variable.class
#=> Float
```

This could be fixed in a subsequent PR if this is useful.

Reply:
I actually thought about something like this, but didn't want to make it confusing and potentially hint that the name of the variables has anything to do with their type. What do you think about naming them `numeric_variable_one` (the integer) and `numeric_variable_two` (the float) ?

Reply:
I thin just moving the `b` down makes it clear.

```suggestion
a = 1
a.class
#=> Integer

b = 1.0
b.class
#=> Float
```